README.rst

```diff
@@ -74,11 +74,27 @@ The easiest way to install Nansat on a Linux machine is to use
     # update miniconda packages and metadata
     conda update -q --yes conda
```

Comment:
I would remove all instructions how to install conda. This is very linux focused.
Can we write:


The easiest way to install Nansat on any platform is to use Anaconda [link to http://docs.continuum.io/anaconda/index] (download installer [https://conda.io/miniconda.html]).
```
# create conda environment and install Nansat from the conda-forge channel
conda create -n py3nansat -c conda-forge nansat
```

Maybe we can move this line to the Usage section?

# activate the environment
source activate py3nansat

Reply:
done